### ISSUE TITLE
Maybe a small error in Notebook ''Multilayer Perceptrons''?

### ISSUE THREAD

**Issue by LinglanZhao** [GokuMohandas/practicalAI#132]
In the cell of **"Training"** in Notebook **''Multilayer Perceptrons''**, the sentence _"6. Repeat steps 2 - 4 until model performs well."_ should be changed into _"6. Repeat steps 2 - **5** until model performs well."_ Because gradient descent is implemented after each iteration.

**Comment by leebeck69** [comment]
How can re reinstall my laptop think it is crupted

**Comment by GokuMohandas** [comment]
hey @LinglanZhao thanks for the fix! and @leebeck69 , sorry for the delay in response. We'll be releasing a much larger and better release this August so look out for that! And it'll all be in TensorFlow 2.0 + Keras and we'll provide complete set up instructions with that.
